mappings/net/minecraft/util/collection/Int2ObjectBiMap.mapping

```diff
@@ -1,12 +1,21 @@
 CLASS net/minecraft/unmapped/C_pkvohekg net/minecraft/util/collection/Int2ObjectBiMap
+	FIELD f_fcagvbpf LOAD_FACTOR F
 	FIELD f_hgbaphoc ids [I
 	FIELD f_hiidchjc EMPTY Ljava/lang/Object;
 	FIELD f_jotgoigl idToValues [Ljava/lang/Object;
 	FIELD f_lgvkwzhs size I
 	FIELD f_poqcloyh values [Ljava/lang/Object;
+	FIELD f_rfzzrczr NOT_FOUND I
 	FIELD f_yyvhlamv nextId I
 	METHOD <init> (I)V
 		ARG 1 size
+	METHOD <init> ([Ljava/lang/Object;[I[Ljava/lang/Object;II)V
+		ARG 1 values
+		ARG 2 ids
+		ARG 3 idToValues
```

Comment:
```suggestion
		ARG 3 valuesByIds
```
to comply with
> When it's enough, name maps based on the value type. Otherwise, name it in the "valuesByKeys" format.

from [CONVENTIONS:Collections](https://github.com/QuiltMC/quilt-mappings/blob/1.18.2/CONVENTIONS.md#collections)